### PR TITLE
MGMT-9424: Add to swagger api to transform cluster from day1 to day2

### PR DESCRIPTION
**Pull request by tsorya**
[MGMT-9424](https://issues.redhat.com/browse/MGMT-9424): Add api to transform cluster from day1 to day2
We already have all the functionality to make transformation from day1 to day2.
This PR is only adding it to swagger

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -5176,6 +5176,49 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/clusters/{cluster_id}/actions/allow-add-nodes:
```
Comment (avishayt) [inline swagger.yaml:5179]: @mhrivnak @eranco74 @filanov Others...
How do you feel about this name?  I thought "transform" was too vague.  Maybe "allow-add-workers" is better?  Other options?

Comment (filanov) [reply swagger.yaml:5179]: @nmagnezi FYI

Comment (filanov) [reply swagger.yaml:5179]: It's a bit confusing to see that we have both node, host and agent not sure what is right but we should consider a single standard

**Review by filanov — CHANGES_REQUESTED** [review]
Please add subsystem test for the rest-api

File: swagger.yaml (hunk 1 above)
Comment (avishayt) [reply swagger.yaml:5179]: So maybe allow-add-workers is better in that sense of avoiding node/host/agent.  Is allow-adding-workers even better?

Comment (tsorya) [reply swagger.yaml]: allow-add-workers sounds the best, changing

File: subsystem/cluster_test.go [hunk 2]
```diff
@@ -2116,6 +2116,15 @@ var _ = Describe("cluster install", func() {
 			}
 		})
 
+		It("Transform installed cluster to day2", func() {
```
Comment (eranco74) [inline subsystem/cluster_test.go:2119]: Bad flows??

Comment (filanov) [reply subsystem/cluster_test.go:2119]: +1

**Pull request merged**